src/backup.ts

```diff
@@ -97,26 +111,36 @@ export type ExistsCallback = NormalCallback<boolean>;
  * const spanner = new Spanner();
  * const instance = spanner.instance('my-instance');
  * const backup = instance.backup('my-backup');
+ * const sourceName = instance.backup('my-source-backup');
```

Comment:
I would add a separate example on how to copy a backup. I think you can have multiple `@example` blocks.